ui.R

```diff
@@ -145,7 +145,8 @@ ui <- dashboardPage(
                                     outline = TRUE,
                                     status = "primary",
                                     icon = icon("check")
-                                )
+                                ),
+                                downloadButton("deadCoralDl", "Download Plot")
```

Comment:
Could you please name this ID based on 'snake case,' which an internal styling standard we have at ERI? All undercase, separated words by underscores. For example:
`coral_health_plot_download`
Once you change it here, you will also have to change it in the server too to match. Thanks!